lib/config/options/index.ts

```diff
@@ -2350,4 +2354,13 @@ function loadManagerOptions(): void {
   }
 }
 
+function setExperimentalFalse(): void {
+  for (const option of options) {
+    if (!option.experimental) {
+      option.experimental = false;
+    }
+  }
+}
+
```

Comment:
I think that the end result would be the same even if  `option.experimental` is undefined.